ush/set_up_verif_global.sh

```diff
@@ -223,6 +283,29 @@ elif [ $machine = "JET" ]; then
     export obdata_dir="/lfs4/HFIP/hfv3gfs/Mallory.Row/obdata"
     export ccpa_24hr_arch_dir="/lfs4/HFIP/hfv3gfs/Mallory.Row/obdata/ccpa_accum24hr"
     export METviewer_AWS_scripts_dir="/lfs4/HFIP/hfv3gfs/Mallory.Row/VRFY/METviewer_AWS"
+elif [ $machine = "GAEAC5" ]; then
+    export NWROOT="/gpfs/f5/${ACCOUNT}/world-shared/global/glopara/data/nwpara"
+    export HOMEDIR="/gpfs/f5/${ACCOUNT}/scratch/${USER}"
+    export STMP="/gpfs/f5/${ACCOUNT}/scratch/${USER}/stmp2"
+    export PTMP="/gpfs/f5/${ACCOUNT}/scratch/${USER}/stmp4"
+    export NOSCRUB="/gpfs/f5/${ACCOUNT}/scratch/${USER}/noscrub"
+    export global_archive="/gpfs/f5/${ACCOUNT}/world-shared/global/metplus.data/archive"
+    export prepbufr_arch_dir="/gpfs/f5/${ACCOUNT}/world-shared/global/metplus.data/prepbufr"
+    export obdata_dir="/gpfs/f5/${ACCOUNT}/world-shared/global/metplus.data/obdata"
+    export ccpa_24hr_arch_dir="/gpfs/f5/${ACCOUNT}/world-shared/global/metplus.data/obdata/ccpa_accum24hr"
+    export METviewer_AWS_scripts_dir="/scratch1/NCEPDEV/global/Mallory.Row/VRFY/METviewer_AWS"
+elif [ $machine = "GAEAC6" ]; then
+    export NWROOT="/gpfs/f6/${ACCOUNT}/world-shared/global/glopara/data/nwpara"
+    export HOMEDIR="/gpfs/f6/${ACCOUNT}/scratch/${USER}"
+    export STMP="/gpfs/f6/${ACCOUNT}/scratch/${USER}/stmp2"
+    export PTMP="/gpfs/f6/${ACCOUNT}/scratch/${USER}/stmp4"
+    export NOSCRUB="/gpfs/f6/${ACCOUNT}/scratch/${USER}/noscrub"
+    export global_archive="/gpfs/f6/${ACCOUNT}/world-shared/global/metplus.data/archive"
+    export prepbufr_arch_dir="/gpfs/f6/${ACCOUNT}/world-shared/global/metplus.data/prepbufr"
+    export obdata_dir="/gpfs/f6/${ACCOUNT}/world-shared/global/metplus.data/obdata"
+    export ccpa_24hr_arch_dir="/gpfs/f6/${ACCOUNT}/world-shared/global/metplus.data/obdata/ccpa_accum24hr"
+    export METviewer_AWS_scripts_dir="/scratch1/NCEPDEV/global/Mallory.Row/VRFY/METviewer_AWS"
+
```

Comment:
Remove these duplicate lines for Gaea C5 and C6 or make the path updates here and remove the Gaea sections above.